<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication2Class</class>
 <widget class="QMainWindow" name="QtWidgetsApplication2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1822</width>
    <height>1141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="FFmpegOpenGL" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>160</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="FFMpegWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>810</y>
      <width>601</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="playBtn">
       <property name="text">
        <string>播放</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playBtn2">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopBtn">
       <property name="text">
        <string>暂停</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>770</y>
      <width>173</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>0.0/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>780</y>
      <width>173</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="video1_label_1">
       <property name="text">
        <string>0.0/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="video1_label_2">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FFMpegWidget</class>
   <extends>QWidget</extends>
   <header>ffmpegwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FFmpegOpenGL</class>
   <extends>QOpenGLWidget</extends>
   <header>ffmpegopengl.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtWidgetsApplication2.qrc"/>
 </resources>
 <connections/>
</ui>
